addons/website/static/src/systray_items/new_content.js

```diff
@@ -131,10 +138,16 @@ export class NewContentModal extends Component {
     async onWillStart() {
         this.isDesigner = await this.user.hasGroup('website.group_website_designer');
         this.canInstall = await this.user.isAdmin;
-
-        const xmlIds = this.state.newContentElements.filter(({status}) => status === MODULE_STATUS.NOT_INSTALLED).map(({moduleXmlId}) => moduleXmlId);
         if (this.canInstall) {
-            this.modulesInfo = await this.rpc('/website/get_modules_info', {xml_ids: xmlIds});
+            const moduleNames = this.state.newContentElements.filter(({status}) => status === MODULE_STATUS.NOT_INSTALLED).map(({moduleName}) => moduleName);
+            this.modulesInfo = {};
+            for (const record of await this.orm.searchRead(
+                "ir.module.module",
```

Comment:
Nitpicking but try to use single quote for non literal string, even weirder here as next line you are using single quote